data/examples/declaration/signature/minimal/minimal.hs

```diff
@@ -0,0 +1,6 @@
+
+  {-# MINIMAL     (==)   |(  (/=) ,       foo) #-}
+
+  {-#    MINIMAL a | (b , c      , d | e
```

Comment:
What is this syntax with parentheses?

Reply:
OK, I think I understand how the syntax, although I've never seen it used.

Reply:
I think it's just to enforce precedence, otherwise **and** has a higher precedence over **or** and you can't enforce custom minimal definition rules.